Quaternion/Quaternion.py

```diff
@@ -479,6 +550,9 @@ def __mul__(self, quat2):
 
           q = Quat(numpy.dot(q1.transform, q2.transform))
```

Comment:
Might as well update this to the modern nicer version `q = Quat(q1.transform @ q2.transform)` (but your caveat definitely applies).

Reply:
I think numpy.dot is not correct here. It should be as you say or numpy.matmul.

Also, docstrings should use the new API. So this should be q = Quat(transform=q1.transform @ q2.transform)

Reply:
Agreed, dot only works in the scalar case where the transforms are 3x3.